### ISSUE TITLE
FNX3-16644 ⁃ [Crash] Crash at forced sync

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#2876]
## Steps to reproduce
1. Log in with Fxa.
1. Go to Settings>Account, tap Sync now.
    1. Expected behavior
FxA starts syncing.
    1. Actual behavior
App crashes
    1. Device information

* Android device: Note 8 (Android 9), Nexus 5 (Android 6)
* Fenix version: 5/27, build #11470614

**Comment by jonalmeida** [comment]
Found the crash log for this issue:
```
2019-05-27 10:16:59.931 6879-6879/org.mozilla.fenix.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.debug, PID: 6879
    java.lang.ClassCastException: androidx.preference.EditTextPreference cannot be cast to androidx.preference.CheckBoxPreference
        at org.mozilla.fenix.settings.AccountSettingsFragment.updateSyncingItemsPreference(AccountSettingsFragment.kt:202)
        at org.mozilla.fenix.settings.AccountSettingsFragment.access$updateSyncingItemsPreference(AccountSettingsFragment.kt:37)
        at org.mozilla.fenix.settings.AccountSettingsFragment$syncStatusObserver$1$onStarted$1.invokeSuspend(AccountSettingsFragment.kt:159)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:238)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by sv-ohorvath** [comment]
Verified as fixed on 5/28 (build #11471807)
